src/OSPSuite.SimModel/src/Simulation.cpp

```diff
@@ -1378,8 +1378,12 @@ std::string Simulation::GetSimulationXMLString ()
 		for(idx=0; idx<_parameters.size(); idx++)
 			_parameters[idx]->UpdateFormulaInXMLNode(formulaListNode, parametersListNode);
 
-		for(idx=0; idx<_species.size(); idx++)
-			_species[idx]->UpdateFormulaInXMLNode(formulaListNode, speciesListNode);
+      for (idx = 0; idx < _species.size(); idx++)
+      {
+         _species[idx]->UpdateFormulaInXMLNode(formulaListNode, speciesListNode);
+         //also update the scale factor in the XML node.
+         _species[idx]->UpdateScaleFactorInXMLNode(speciesListNode);
+      }
```

Comment:
Could you also please change indentation in your code text editor? So that the new code will have the same indentation as the old one.

Reply:
Coding standars?

Visual Studio Settings

Identation of 3
Use spaces instead of tabs